tests/unit/test_linear_learner.py

```diff
@@ -216,7 +216,7 @@ def test_num_classes_can_be_string_for_multiclass_classifier(sagemaker_session):
     LinearLearner(sagemaker_session=sagemaker_session, **test_params)
 
 
-@pytest.mark.parametrize("iterable_hyper_parameters, value", [("eval_metrics", 0)])
+@pytest.mark.parametrize("iterable_hyper_parameters, value", [("max_iterations", [0])])
```

Comment:
was this test not working before?

Reply:
It was working before. The test expects throwing a TypeError that ``eval_metrics`` should be a list instead of an integer.